decidim-forms/app/models/decidim/forms/display_condition.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Forms
+    # The data store for a DisplayCondition in the Decidim::Forms component.
+    class DisplayCondition < Forms::ApplicationRecord
+      enum condition_type: [:answered, :not_answered, :equal, :not_equal, :match], _prefix: true
+
+      belongs_to :question, class_name: "Question", foreign_key: "decidim_question_id", inverse_of: :display_conditions
+      belongs_to :condition_question, class_name: "Question", foreign_key: "decidim_condition_question_id", inverse_of: :display_conditions_for_other_questions
+      belongs_to :answer_option, class_name: "AnswerOption", foreign_key: "decidim_answer_option_id", optional: true
```

Comment:
A display condition will always belong to a :question and sometimes also can belong to an :answer_option?
Also what's the role of a condition_question? should we document it here or in Question class?
Can we document it in the signature of this class?